src/sentry/integrations/slack/notify_action.py

```diff
@@ -47,7 +48,7 @@ def __init__(self, *args, **kwargs):
 
     def clean(self):
         channel_id = None
-        if self.data.get("input_channel_id"):
+        if self.data.get("inputChannelId") or self.data.get("input_channel_id"):
```

Comment:
I recognize supporting both is sort of weird but based on the description of https://github.com/getsentry/sentry/pull/20338 it sounds like users are already passing the snake case, but _not_ allowing people to pass the camel case felt weird, so I just did both.